test/unit/omp-target/test-nested-reduce.cpp

```diff
@@ -17,28 +17,29 @@ static RAJA::Index_type const xExtent = 64;
 static RAJA::Index_type const yExtent = 64;
 static RAJA::Index_type const area = xExtent * yExtent;
 
-TEST(NestedReduceTargetOMP, outer)
-{
-  RAJA::Index_type l_begin = begin;
-  RAJA::Index_type l_xExtent = xExtent;
-  RAJA::ReduceSum<RAJA::omp_target_reduce, double> sumA(0.0);
-  RAJA::ReduceMin<RAJA::omp_target_reduce, double> minA(10000.0);
-  RAJA::ReduceMax<RAJA::omp_target_reduce, double> maxA(0.0);
-  RAJA::RangeSegment xrange(begin, xExtent);
-  RAJA::RangeSegment yrange(begin, yExtent);
-
-  RAJA::forall<RAJA::omp_target_parallel_for_exec<64>>(yrange, [=](int y) {
-    RAJA::forall<RAJA::seq_exec>(xrange, [=](int x) {
-      sumA += double(y * l_xExtent + x + 1);
-      minA.min(double(y * l_xExtent + x + 1));
-      maxA.max(double(y * l_xExtent + x + 1));
-    });
-  });
-
-  ASSERT_FLOAT_EQ((area * (area + 1) / 2.0), sumA.get());
-  ASSERT_FLOAT_EQ(1.0, minA.get());
-  ASSERT_FLOAT_EQ(area, maxA.get());
-}
+// TEST(NestedReduceTargetOMP, outer)
```

Comment:
We'll convert this to a RAJA::kernel implementation when we get to reworking those tests.

Reply:
That's what it should be yeah, the current version can't work until we do that swap.